sql/structured.go

```diff
@@ -135,6 +135,11 @@ func (desc *TableDescriptor) SetName(name string) {
 	desc.Name = name
 }
 
+// IsSystem returns true if this is a system table.
+func (desc *TableDescriptor) IsSystem() bool {
+	return desc.GetID() == namespaceTableID || desc.GetID() == descriptorTableID
```

Comment:
I think it would be better to check if the id is in the range `[1,MaxReservedID]`.